crawler/src/utils/utils.ts

```diff
@@ -92,3 +94,12 @@ export const resolvePromisesAsChunks = async <T>(
 };
 
 export const removeUndefinedItem = <Type, >(item: (Type|undefined)): item is Type => item !== undefined;
+
+export const toContractAddress = (address: string): string =>{
```

Comment:
use the one you wrote in API much cleaner 
```
export const toContractAddress = (address: string): string =>
  utils.getAddress(address.trim().toLowerCase());
```

Reply:
want to catch error here so we don't stop the explorer